.github/ISSUE_TEMPLATE.md

```diff
@@ -9,6 +9,7 @@
 [//]: # (Leave in the line that best describes the issue you are reporting.)
 
 Type: ruleset issue
+Type: ruleset enhancement
```

Comment:
I think @jeremyn's comment in https://github.com/EFForg/https-everywhere/issues/13890#issuecomment-351230054 is fair: we should probably reduce the overhead and only have a `ruleset issue` and `new ruleset` sub-tier label, both of which would get the `ruleset` label.

Reply:
@Hainish whoops, this fell off my radar somehow. Fixed.